aerial_autonomy_stacks.md

```diff
@@ -75,3 +75,19 @@ Working list:
 A list of packages which don't comprise a full stack but do offer value on top of basic flight controller firmware.
 
 * [MAVROS Controllers](https://github.com/Jaeyoung-Lim/mavros_controllers)
+
+## Indoor navigation packages
+ROS compatible indoor navigation packages that were developed and tested on aerial robotic platforms are listed below. The list, though not exhaustive, provides a good overview of off-the-shelf indoor navigation software. 
```

Comment:
Perhaps it would be good to mention on which this comparison differs compared to the table given on top of this document?

Reply:
How about:  "Given the above listed [Aerial Autonomy Stacks](https://google-cartographer-ros.readthedocs.io/en/latest), the software list below outlines specific implementations of indoor navigation packages in ROS, running on aerial vehicle platforms. The list, though not exhaustive, provides a good overview of off-the-shelf non-commercial indoor navigation software."

Reply:
that sounds good to me :)